astropy/coordinates/earth.py

```diff
@@ -555,12 +571,11 @@ def _get_site_registry(cls, force_download=False, force_builtin=False):
                     if force_download:
                         raise
                     msg = (
-                        "Could not access the online site list. Falling "
-                        "back on the built-in version, which is rather "
-                        "limited. If you want to retry the download, do "
-                        "{0}._get_site_registry(force_download=True)"
+                        "Could not access the main site list. Falling back on the "
+                        "built-in version, which is rather limited. If you want to "
+                        "retry the download, use the option `refresh_cache=True`."
```

Comment:
I don't think single backticks do anything here since this warning would be issued in plain text?
```suggestion
                        "retry the download, use the option refresh_cache=True."
```

Reply:
Single quote might be easier on the eyes in plain text. At least that is my personal opinion. I am not fonts expert.